node/countries/rules.ts

```diff
@@ -0,0 +1,14 @@
+import { ISOAlpha3 } from './types'
+import ARG from './ARG'
+import BOL from './BOL'
+import BRA from './BRA'
+import USA from './USA'
+
+const rules: { [key in ISOAlpha3]?: Rules } = {
```

Comment:
I think it would be best to pull in this rules from [`country-data-settings`](https://github.com/vtex-apps/country-data-settings) instead of inlining them here.

Reply:
@lucasecdb, how about doing this in another PR/task? So we can move the rules from `places-graphql` as well